### ISSUE TITLE
Error in seq_len(no) : argument must be coercible to non-negative integer

### ISSUE THREAD

**Issue by pfgherardini** [igraph/rigraph#104]
Hi all,

I am getting the same issue as described in this thread.

http://lists.nongnu.org/archive/html/igraph-help/2015-07/msg00055.html

I uploaded to Dropbox a graphml file to reproduce the issue. This file loads correctly in older versions of igraph.

https://www.dropbox.com/s/t8kp7ds9t0shvt5/test.graphml?dl=0

Steps to reproduce

```
library(igraph)
G <- read.graph("test.graphml", format = "graphml")
G
```

on my machine this produces

```
IGRAPH UNWB 220 425 -- 
+ attr: name (v/c), x (v/n), y (v/n), cellType (v/n), Time (v/n), Cell_length (v/n), BC1 (v/n),
| BC2 (v/n), BC3 (v/n), BC4 (v/n), BC5 (v/n), BC6 (v/n), Ter119 (v/n), CD45.2 (v/n), Ly6G (v/n),
| IgD (v/n), CD11c (v/n), F480 (v/n), CD3 (v/n), NKp46 (v/n), CD23 (v/n), CD34 (v/n), CD115
| (v/n), CD19 (v/n), 120g8 (v/n), CD8 (v/n), Ly6C (v/n), CD4 (v/n), CD11b (v/n), CD27 (v/n),
| CD16_32 (v/n), SiglecF (v/n), Foxp3 (v/n), B220 (v/n), CD5 (v/n), FceR1a (v/n), TCRgd (v/n),
| CCR7 (v/n), Sca1 (v/n), CD49b (v/n), cKit (v/n), CD150 (v/n), CD25 (v/n), TCRb (v/n), CD43
| (v/n), CD64 (v/n), CD138 (v/n), CD103 (v/n), IgM (v/n), CD44 (v/n), MHCII (v/n), DNA1 (v/n),
| DNA2 (v/n), Cisplatin (v/n), beadDist (v/n), Label (v/c), type (v/n), groups (v/n), sample
| (v/c), popsize (v/n), r (v/n), g (v/n), b (v/n), size (v/n), id (v/c), highest_scoring_edge
| (v/n), desc (v/c), weight (e/n), r (e/n), g (e/n), b (e/n), edge_type (e/c)
+ edges (vertex names):
Error in seq_len(no) : argument must be coercible to non-negative integer
```

sessionInfo

```
R version 3.2.0 (2015-04-16)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.9.4 (Mavericks)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] igraph_1.0.1

loaded via a namespace (and not attached):
[1] magrittr_1.5 tools_3.2.0 
```

**Comment by swainjo** [comment]
Any update on this, I get the same error. Happy to provide further information about my system/files etc if required.

**Comment by cchan** [comment]
I have the same error as well. Windows, RStudio.

**Comment by mkoohafkan** [comment]
In `print.igraph`, the call to `head_lines` will sometimes return a value greater than the `max.lines` argument. This results in an error in `.print.edges.compressed.limit` where `seq_len` is passed a negative number (`max.lines - head_lines`). 

Thankfully, this error seems to just be a printing issue and does not indicate problems with the igraph data structure itself.

**Comment by ntamas** [comment]
This should now be fixed in https://github.com/igraph/rigraph/commit/e98726724d77fc80cad707c4e56f19412c31bcaa . The fix will be released in 1.3.0. Feel free to reopen if the issue still persists after installing 1.3.0 or the current development version.
